home-screen-widget/profiles/DEVELOPERS_PORTAL.md

```diff
@@ -0,0 +1,412 @@
+
+
+E.g.
+
+```json
+{
+  "type": "shortcuts",
+  "url": "/home-screen-core-widgets/js/streams.js",
+  "featureFlags": [streams],
+  "propertiesSchema": {
+    "pinnedStreams": {
+      "$id": "#/propertiesSchema/pinnedStreams",
+      "default": [],
+      "description": "Indicates which streams are pinned",
+      "examples": [
+        [
+          654321,
+          123456
+        ]
+      ],
+      "items": {
+        "$id": "#/propertiesSchema/pinnedStreams/items",
+        "default": -1,
+        "description": "Refers to pinned stream id",
+        "examples": [
+          654321,
+          123456
+        ],
+        "title": "Pinned stream id",
+        "type": "integer"
+      },
+      "title": "Pinned streams schema",
+      "type": "array"
+    }
+  },
+  "_mimeType": "application/vnd.io.beekeeper.home-screen.configuration.widget-type.v1+json",
+  "_self": {
+    "href": "/api/2/home-screen/widget-types/shortcuts",
+    "method": "GET"
+  },
+}
+
+```
+
+#### Widgets Configuration
+A widget configuration determines how a widget type is configured for a specific tenant. For every widget type there can be multiple widget configurations for various tenants. Also a tenant can have multiple configurations for the same widget but with different access control settings, e.g. the same widget differently configured for two different groups.
+
+For instance a simple widget configuration: 
+
+```json
+{
+    "typeName": "streams",
+    "properties": {
+        "pinnedStreams": [876543, 321, 345678]
+    },
+    "acl":{}
+}
+```
+
+#### Home screen configuration
+The home screen can be configured on a tenant basis. Configuring the home screen means specifying which widgets will appear in the home screen and in which order for a specific tenant.
+
+
+For instance a widget configuration containing the configured streams and the shortcuts widgets could look like this: 
+
+
+```json
+{
+  "widgets": [
+    {
+      "_mimeType": "application/vnd.io.beekeeper.home-screen.configuration.widget.v1+json",
+      "_operations": {...},
+      "_relations": {...},
+      "_self": {},
+      "id": "b5575b50-0fb9-44fd-a635-85622f0061d1",
+      "properties": {},
+      "typeName": "shortcuts",
+      "url": "/home-screen-core-widgets/js/shortcuts.js"
+    },
+    {
+      "_mimeType": "application/vnd.io.beekeeper.home-screen.configuration.widget.v1+json",
+      "_operations": {...},
+      "_relations": {...},
+      "_self": {...},
+      "id": "39261cdd-0d40-4649-bc96-7c8cd251ea0b",
+      "properties": {},
+      "typeName": "streams",
+      "url": "/home-screen-core-widgets/js/streams.js"
+    }
+  ],
+  "_mimeType": "application/vnd.io.beekeeper.home-screen.configuration.v1+json",
+  "_operations": {...},
+  "_self": {...}
+}
+
+```
+
+
+### Access management (ACL)
+Widgets access can be modified in order t
+
+#### Fetching the acl
+Every entity has acl associcated to it. These can be accessed by following the acl url in `rels` in order to get them
```

Comment:
missing an article for "acl".

Also, in general I would recommend to write all acronyms (ID, URL, ACL, SDK, API) in all caps.

Reply:
Missing fullstop